### DISCUSSION TITLE
Siege Boot Bypass

### DISCUSSION THREAD

**Discussion by CeIebrimbor** [GriefPrevention/GriefPrevention#425] [Ideas]
I know, another request for Siege...this one is easy (I think).

When a Sieger wins, they gain access to doors n' such. After `DoorsOpenDelayInSeconds`, that access is removed and the Sieger(s) are booted from the claim, I assume with the same process as /trapped, but by force.

The issue, if the Sieger(s) log out and the time expires, they bypass the boot from the claim. They log in and are still inside the claim.

**The Request:**
Run the `DoorsOpenDelayInSeconds` boot on schedule, but also if the Sieger(s) log out during that timeframe.

**Comment by RoboMWM** [comment]
moving to discussions (ideas category) hence the temp label
